src/main/kotlin/io/github/mojira/arisa/ModuleRegistry.kt

```diff
@@ -415,7 +258,11 @@ class ModuleRegistry(jiraClient: JiraClient, private val config: Config) {
                             defaultMessage
                         ) else defaultMessage
 
-                    addRestrictedComment(issue, text, "helper")
+                    addRestrictedComment(
+                        issue,
+                        text,
+                        "helper"
+                    )
```

Comment:
Imo we can just leave this as-is. I plan to refactor the adding comment part when resolving #139; this whole part could be replaced by a single `partially` function reference at that time.